### PR TITLE
Fix Installation in README.md

### PR DESCRIPTION
**Pull request by nowsprinting**
### Changes
- Mod **Installation** section and add screenshots
- Add **How to development** section
- Fix wording. Apply [it-english-style-guide](https://styleguide.progeigo.org/docs/intro/)

---

### Contribution License Agreement

- [x] By placing an "x" in the box, I hereby understand, accept and agree to be bound by the terms and conditions of the [Contribution License Agreement](https://dena.github.io/cla/).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -18,47 +18,49 @@ Click [日本語](./README_ja.md) for the Japanese page if you need.
...
-```
+1. Open the **Package Manager** tab in Player Settings window (**Editor > Player Settings**)
+2. Click **+** button under the **Scoped Registries** and enter the following settings (figure 1.):
+   1. **Name:** `package.openupm.com`
+   2. **URL:** `https://package.openupm.com`
+   3. **Scope(s):** `dom.dena`, `com.cysharp`, and `com.nowsprinting`
+3. Open the Package Manager window (**Window > Package Manager**) and select **My Registries** in registries drop-down list (figure 2.)
+4. Click **Install** button on the `com.dena.anjin` package
+
+**Fugure 1.** Package Manager tab in Player Settings window.
```
Comment (get-me-power) [inline README.md:33]: Figure が正しいかと思います

File: README.md [hunk 2]
```diff
@@ -18,47 +18,49 @@ Click [日本語](./README_ja.md) for the Japanese page if you need.
...
+
+![](Documentation~/ProjectSettings_Dark.png#gh-dark-mode-only)
+![](Documentation~/ProjectSettings_Light.png#gh-light-mode-only)
+
+**Fugure 2.** Select registries drop-down list in Package Manager window.
```
Comment (get-me-power) [inline README.md:38]: 同じく

Comment (nowsprinting) [reply README.md]: 同上

**Review by get-me-power — APPROVED** [review]
LGTM!

**Pull request merged**